components/molecule/selectPopover/src/index.js

```diff
@@ -30,6 +30,7 @@ const MoleculeSelectPopover = ({
   fullWidth,
   hideActions,
   iconArrowDown: IconArrowDown,
+  id = '',
```

Comment:
is it really required an empty string?
```suggestion
  id,
```

Reply:
Not really, but I think it adds consistency throughout the component when any developer wants to work/iterate with `id`

Reply:
do you consider it unnecessary now?